src/sagemaker/serve/utils/telemetry_logger.py

```diff
@@ -62,11 +63,13 @@ def wrapper(self, *args, **kwargs):
             caught_ex = None
 
             image_uri_tail = self.image_uri.split("/")[1]
+            image_uri_option = _get_image_uri_option(self.image_uri, self._is_custom_image_uri)
             extra = (
                 f"{func_name}"
                 f"&x-modelServer={MODEL_SERVER_TO_CODE[str(self.model_server)]}"
                 f"&x-imageTag={image_uri_tail}"
                 f"&x-sdkVersion={SDK_VERSION}"
+                f"&x-defaultImageUsage={image_uri_option}"
```

Comment:
curious, is "default" important to the naming here or can we just go with "imageUsage"

Reply:
or "imageType"

Reply:
I think so, this metric is to capture whether customers provide their own image or not, the further differentiation on custom image or custom 1p image is not that important.

Reply:
ack sound good. posed the question from a string len perspective. 1024 is max fyi